### PR TITLE
opus 1.3.1

### PR DESCRIPTION
**Pull request by ericLemanissier**
Specify library name and version:  **opus/1.3.1**

- [X] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [X] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [X] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [X] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/opus/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    default_options = {'shared': False, 'fPIC': True, 'fixed_point': False}
+
+    _source_subfolder = "source_subfolder"
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+        if self.settings.os == "Windows" and \
+                (self.settings.compiler == "Visual Studio" and int(str(self.settings.compiler.version)) < 14):
```
Comment (uilianries) [inline recipes/opus/all/conanfile.py]:
```suggestion
                self.settings.compiler == "Visual Studio" and Version(self.settings.compiler.version) < "14":
```

File: recipes/opus/all/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
+from conans import ConanFile, tools, CMake
```
Comment (uilianries) [inline recipes/opus/all/conanfile.py]:
```suggestion
from conans import ConanFile, tools, CMake
from conans.tools import Version
```

File: recipes/opus/all/FindOPUS.cmake [hunk 3]
```diff
@@ -0,0 +1,18 @@
+find_path(
```
Comment (uilianries) [inline recipes/opus/all/FindOPUS.cmake]: Why do you need this?

File: recipes/opus/all/opus_buildtype.cmake [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# Set a default build type if none was specified
```
Comment (uilianries) [inline recipes/opus/all/opus_buildtype.cmake]: Why do you need this if Conan by default sets Release?

File: recipes/opus/all/FindOPUS.cmake (hunk 3 above)
Comment (ericLemanissier) [reply recipes/opus/all/FindOPUS.cmake]: I don't know, I took bincrafters' recipe "as is". I'll do some cleanup

File: recipes/opus/all/opus_buildtype.cmake (hunk 4 above)
Comment (ericLemanissier) [reply recipes/opus/all/opus_buildtype.cmake]: I have no idea, bincrafters' recipe legacy again.

Comment (ericLemanissier) [reply recipes/opus/all/opus_buildtype.cmake]: this whole file is working around an upstream bu: https://github.com/xiph/opus/issues/129

**Comment by uilianries** [comment]
Thanks @ericLemanissier !

**Pull request merged**